doc/interop-test-descriptions.md

```diff
@@ -1391,22 +1404,38 @@ fully communicate metadata.
 ### Backend metrics report
 [Backend Metrics Report]: #backend-metrics-report
 
-Server reports backend metrics data in both per-query and out-of-band cases, with metrics data
-indicated from the unary call request.
+#### Report per-query metrics
+
+Server reports backend metrics data in per-query case, echoing metrics data from the unary call request.
 
 Using ORCA APIs we install the per-query metrics reporting server interceptor, so that it can attach 
 metrics per RPC.
-Also using ORCA APIs we register the `OpenRCAService` implementation to the server, so that it can
-report metrics periodically. The minimum report interval in the ORCA service is set to 1 sec.
 
-During test, the server will receive unary requests from the client that each may contain up to two
-test load report, indicating whether it needs to update metrics for the current call or at the OOB server.
+During `orca_per_query` test, the server will receive unary requests from the client that may contain a 
+test load report, indicating whether it needs to update metrics for the current call.
 Then the server sets the metrics data, echoing the test load report it just received.
-Specifically:
-1. If the `orca_per_rpc_report` is set in the request, the server test driver will call
+Specifically, if the `orca_per_rpc_report` is set in the request, the server test driver will call
 `CallMetricRecorder` to record both utilization and 
 request cost metrics for the current RPC, the metrics data is coped from the received test load 
 report from the request.
-2. And if the `orca_oob_report` is set in the request, the server test driver will call
-`MetricRecorder` to first clear all the previous metrics data, and then record the utilization 
-metrics, the metrics data is coped from the received test load report from the request.
+
+#### Report out-of-band metrics
+Server reports backend metrics data in out-of-band case, echoing metrics data from full duplex
+call request.
+
+Using ORCA APIs we register the `OpenRCAService` implementation to the server, so that it can
+report metrics periodically. The minimum report interval in the ORCA service is set to 1 sec.
+
+The server implementation should use a lock or similar mechanism to allow only one client to perform the `orca_oob` test at a time
+in order to not interfere with other clients' tests.
+
+During `orca_oob` test, the server receives full duplex requests from the client that contains a
+test load report, indicating whether it needs to update metrics at the OOB server.
+When the `orca_oob_report` is set in the request, the server test driver will block until
+it acquires the oob lock. If the stream wins the lock, the server calls `MetricRecorder` to first clear 
```

Comment:
"When the stream acquires". There's no "if"; it is only a question of "when." "If" should make you wonder, "what if this doesn't happen?" A stream "winning" the lock makes it sound like winning a race or the like, and definitely sounds like it can lose.